discoveryengine/import_documents_sample.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2023 Google LLC
+# Copyright 2024 Google LLC
```

Comment:
Please don't update copyright years! They should reflect the years the file is introduced, not the current date.

The only exception to this is autogenerated content by robots, not by humans.